dashboard/app/models/pd/workshop.rb

```diff
@@ -183,6 +183,10 @@ def self.in_state(state)
     end
   end
 
+  def course_name
+    course == Pd::Workshop::COURSE_ADMIN ? 'Administrator' : course
+  end
+
```

Comment:
This seems hacky.

If we want these to be called `Administrator` instead of `Admin`, shouldn't that be across the board? We can change the constant value and upgrade existing data in a migration (which should be a separate PR, run before this).